### DISCUSSION TITLE
Combining VOCABS for training and inference

### DISCUSSION THREAD

**Discussion by mohamedsaeed8223** [mindee/doctr#1818] [Q&A] [answered]
I was wondering if this is feasible at all or not
I tried creating a new vocab the same way the multilingual vocab was written
here's a snippet:
```
VOCABS["englisharabic"] = "".join(
    dict.fromkeys(
        VOCABS["english"]
        + VOCABS["arabic"]
        + "§"
    )
)
```
I added this in the ``` doctr/datasets/vocabs.py``` in my cloned repo
and tried to train a model using it but the train script didn't recognize it
I was wondering if I should add my new vocab to the package code in my env instead or if it isn't doable at all

**Comment by mohamedsaeed8223** [comment]
update: I updated the ```vocabs.py``` in the path ```miniconda3/envs/doctr/lib/python3.10/site-packages/doctr/datasets``` with my new vocab and the training initiated 😅

**Reply by felixdittrich92** [reply, marked as answer]
Hi @mohamedsaeed8223 :wave:,

You can install the repo as `editable` then you can modify the vocab directly in you IDE

```
cd doctr
pip3 install -e .[torch]  # for example
```
